client/executor/common/src/runtime_blob/runtime_blob.rs

```diff
@@ -81,6 +81,23 @@ impl RuntimeBlob {
 		export_mutable_globals(&mut self.raw_module, "exported_internal_global");
 	}
 
+	/// Perform an instrumentation that makes sure that a specific function `entry_point` is exported
+	pub fn entry_point_exists(blob: &RuntimeBlob, entry_point: &str) -> bool {
```

Comment:
```suggestion
	pub fn entry_point_exists(&self, entry_point: &str) -> bool {
```

Reply:
Yes, I am adding the invocation of this verification in the prevalidate function in the host.rs of the pvf crate.